hack/sanity.sh

```diff
@@ -19,8 +19,8 @@ source "${script_dir}"/common.sh
 setGoInProw $GOLANG_VER
 
 echo "docker repo: [$DOCKER_REPO]"
-go test -o _out/sanity.test -c -v ./sanity/...
-docker build -t ${DOCKER_REPO}/sanity:test -f ./sanity/Dockerfile .
+CGO_ENABLED=0 go test -o _out/sanity.test -c -v ./sanity/...
+buildah build -t ${DOCKER_REPO}/sanity:test -f ./sanity/Dockerfile .
 # Need privileged so we can bind mount inside container, and hostpath capacity cannot change, so skipping that test
-docker run --privileged ${DOCKER_REPO}/sanity:test -ginkgo.noColor -ginkgo.skip="should fail when requesting to create a volume with already existing name and different capacity"
+podman run --privileged ${DOCKER_REPO}/sanity:test -ginkgo.noColor -ginkgo.skip="should fail when requesting to create a volume with already existing name and different capacity"
```

Comment:
Looks like podman is not installed in this container, this is why the test is failing here.

Reply:
quay.io/kubevirtci/golang is the container that is running this test. Might have to install podman on it during this run.

Reply:
After a failed attempt to make the CI image include podman (the kubevirtci kind provider would pick it up but then fail to come up, so it immediately broke some CI lanes), this PR now supports both docker & podman, so CI is now passing with no change to project-infra.